lib/update_comfyui.py

```
@@ -231,13 +235,47 @@ def main():
+    pre_reset_head = repo.head.target
+    was_detached = repo.head_is_detached
+    pre_master_target = branch.target if branch is not None else None
+    try:
+        if branch is None:
+            repo.create_branch("master", repo.get(remote_id))
+        else:
+            branch.set_target(remote_id)
+        ref = repo.lookup_reference("refs/heads/master")
+        repo.checkout(ref, strategy=pygit2.GIT_CHECKOUT_FORCE)
+        repo.reset(remote_id, pygit2.GIT_RESET_HARD)
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)